soc/xtensa/nxp_adsp/CMakeLists.txt

```
@@ -1,20 +1,26 @@
-# NXP i.MX8 SoC family CMake file
+# NXP i.MX8/RT SoC family CMake file
 #
 # Copyright (c) 2021 NXP
 # SPDX-License-Identifier: Apache-2.0
 
+if(CONFIG_SOC_SERIES_NXP_RT595)
```

```suggestion
if(CONFIG_SOC_NXP_RT595)
```
Since these macro is for the SOC part number, this Kconfig should be the SOC name, not the SOC series.